### PR TITLE
Update commands for modules rework

### PR DESCRIPTION
**Pull request by napstr**
Also adjusted one or the other thing I noticed to be missing/outdated

This contains documentation for https://github.com/Frederikam/FredBoat/pull/395

### CODE REVIEW

File: public/markdown/index.md [hunk 1]
```diff
@@ -1,52 +1,59 @@
 # Welcome to FredBoat Docs
-FredBoat is a free Discord music bot that delivers high-quality music to your Discord server. The music bot is made to easily plug in to your Discord server with no configuration. 
+FredBoat is a free Discord music bot that delivers high-quality music to your Discord server. FredBoat can be easily plugged into your Discord server with no configuration.
 
 
 ## Setting up the bot
-FredBoat is made to be super simple to use! Getting FredBoat running on your server is easy. You can invite the bot by authenticating it with your account using [this](https://goo.gl/cFs5M9) link. Bear in mind that you must have `Manage Server` permissions to be able to do this.
+FredBoat is made to be super simple to use! Getting FredBoat running on your server is easy. You can invite FredBoatd by authenticating it with your account using [this](https://goo.gl/cFs5M9) link. Bear in mind that you must have `Manage Server` permissions to be able to do this.
```
Comment (Frederikam) [inline public/markdown/index.md]: You left a typo here

**Comment by Frederikam** [comment]
Otherwise this looks good

File: public/markdown/non_music.md [hunk 2]
```diff
@@ -1,39 +1,83 @@
 # Non-music Commands
 
+You can always learn more about any command by saying `;;help ***command***`.
+
+##Configuration
```
Comment (Frederikam) [inline public/markdown/non_music.md]: Doesn't render properly. Add a space after ##

File: public/markdown/non_music.md [hunk 3]
```diff
@@ -1,39 +1,83 @@
...
+| ;;catgirl                  | Post a catgirl.                                                       | ;;neko                                      |
+| ;;facedesk                 | Post a random facedesk gif.                                           | ;;facedesk                                  |
+| ;;hug ***@username***      | Hug someone.                                                          | ;;hug @Shredder121                          |
+| ;;pat ***@username***      | Pat someone.                                                          | ;;pat @Real KC                              |
+| ;;roll                     | Roll around.                                                          | ;;roll                                      |
+| ;;wombat                   | Post a wombat.                                                        | ;;wombat                                    |
 
-## Utility
+
+##Information
```
Comment (Frederikam) [inline public/markdown/non_music.md]: Also this

File: public/markdown/selfhosting.md [hunk 4]
```diff
@@ -223,3 +223,10 @@ wget https://ci.fredboat.com/guestAuth/repository/download/FredBoat_Build/.lastS
 wget https://ci.fredboat.com/guestAuth/repository/download/FredBoat_Build/.lastSuccessful/credentials.yaml.example?branch=refs/heads/dev -O credentials.yaml
 
 ```
+
+
+#Bot Administration
```
Comment (Frederikam) [inline public/markdown/selfhosting.md]: And that...

**Comment by napstr** [comment]
Thanks, Fred.

Let's merge this when the actual code is deployed.

**Comment by Frederikam** [comment]
Odd that the CI/CD failed. Let me investigate

**Pull request merged**